webapp/src/containers/components/modals/styles.css

```diff
@@ -109,3 +102,21 @@ table.table thead {
 .add-actions {
   margin-top: 15px;
 }
+
+.validation-warning {
+  color: red;
+}
+
+.listTable th {
+  text-align: center;
+}
+
+.guestButton.active {
+  box-shadow: none;
+  outline: transparent;
```

Comment:
```suggestion
  outline: none;
```
see if this works

Reply:
had conversation with Tanisha and she suggested using transparent to avoid any unnecessary jump in UI